packages/next-swc/crates/next-custom-transforms/tests/loader/auto-cjs/react-prod/input.js

```diff
@@ -0,0 +1,5 @@
+// This file should not import helpers using ESM syntax
+const E = class Foo {
```

Comment:
We also need a output.js for this? maybe rename from `react-prod` to `modern-syntax-cjs`?

Reply:
Oops I forgot pushing some commits 🤣